docs/dictionaries.md

```diff
@@ -19,24 +19,25 @@ syntax for [json-ld](https://w3c.github.io/json-ld-syntax/#the-context):
 ```json
 {
   "@context": {
-    "bg": "https://terms.neurobagel.org/",
+    "nb": "http://neurobagel.org/vocab/",
     "purl": "http://purl.obolibrary.org/obo/",
     "snomed": "http://purl.bioontology.org/ontology/SNOMEDCT/",
-    "cogAtlas": "https://www.cognitiveatlas.org/task/id/"
+    "cogatlas": "https://www.cognitiveatlas.org/task/id/"
   }
 }
 ```
 
 ## Participant identifier
 
-Term from National Cancer Institute Thesaurus
+Term from the Neurobagel vocabulary.
 
-```json hl_lines="5-6"
+```json hl_lines="6-7"
```

Comment:
this is great! What it makes me think is: we probably want a place where we build and temporarily host the docs as they will look once this PR is merged - so you don't have to pull the branch and build locally to review. I believe the technical term is staging. Maybe we can look into getting this for the docs at first as a nice training example. 

Maybe @Remi-Gau has some thoughts on this. There is a [nilearn docs build workflow](https://github.com/nilearn/nilearn/blob/main/.github/workflows/build-docs.yml), not sure if that's what it is used for.

Reply:
where is your doc deployed?

Reply:
Docs get deployed on GH pages atm afaik

Reply:
I'll turn this conversation into an issue here so we can continue

Reply:
OK I will play around with the action on bids starter kit to figure out how to set it up and I can then open PR here.